comprehensive/lms/static/cookieBanner.js

```diff
@@ -51,4 +52,20 @@ oxa = window.oxa || {};
 
 (oxa));
 
-var cookieNotice = new oxa.cookieBanner();
\ No newline at end of file
+var cookieNotice = new oxa.cookieBanner();
+
+// track the user clicks, anchor/button and exclude the LearnMore link on the banner itself
+window.onload = function () {
+    document.onclick = function (event) {
+        event = event || window.event;
+        var target = event.target || event.srcElement;
+
+        while (target) {
+            if ((target instanceof HTMLAnchorElement || target instanceof HTMLButtonElement) && target.id != 'btnPrivacy') {
```

Comment:
oh i see. i misread the code. i first thought the final comparison operator was equality, but it's actually inequality. i understand why it's written this way now, but i'm pretty sure there is another way of doing this. i'll think about it some more and get back to you